go/logic/inspect.go

```diff
@@ -283,6 +280,21 @@ func (this *Inspector) validateBinlogs() error {
 	return nil
 }
 
+// ValidateLogSlaveUpdates checks that binary log log_slave_updates is set. This test is not required when migrating on replica or when migrating directly on master
+func (this *Inspector) ValidateLogSlaveUpdates() error {
```

Comment:
This seems to be an internal func for the logic package, should be `validateLogSlaveUpdates`